assets/js/modules/adsense/components/dashboard/DashboardZeroData.js

```diff
@@ -35,9 +36,9 @@ import SiteSteps from '../common/SiteSteps';
  */
 export default function DashboardZeroData() {
 	return (
-		<div className="mdc-layout-grid mdc-layout-grid--fill">
-			<div className="mdc-layout-grid__inner">
-				<div className="mdc-layout-grid__cell mdc-layout-grid__cell--span-12">
+		<Grid className="mdc-layout-grid--fill">
```

Comment:
Maybe worth converting this `mdc-layout-grid--fill` class to be `fill` property?

Reply:
Sounds good - I've also added the `alignLeft` prop, mapping to `mdc-layout-grid--align-left`, as this is/was used in the `CompatibilityChecks` component:

https://github.com/google/site-kit-wp/blob/d07d8ac284f44925e633d4e84db42bd16002e8fd/assets/js/components/setup/CompatibilityChecks/index.js#L68

Reply:
Done :+1: